pkg/solidity-utils/test/foundry/utils/BaseTest.sol

```diff
@@ -128,14 +155,16 @@ abstract contract BaseTest is Test, GasSnapshot {
         userKeys.push(brokeUserKey);
     }
 
+    // ------------------------------ Helpers ------------------------------
+
     /**
      * @notice Manipulate rates of ERC4626 tokens.
      * @dev It's important to not have a 1:1 rate when testing ERC4626 tokens, so we can differentiate between
      * wrapped and underlying amounts. For certain tests, we may need to override these rates for simplicity.
      */
     function mockERC4626TokenRates() internal virtual {
-        waDAI.inflateUnderlyingOrWrapped(0, 6 * defaultBalance);
-        waUSDC.inflateUnderlyingOrWrapped(23 * defaultBalance, 0);
+        waDAI.inflateUnderlyingOrWrapped(0, 6 * _defaultAccountBalance);
```

Comment:
Consider using the function consistently, instead of the variable directly (as it's internal)?